ops/framework.py

```diff
@@ -457,6 +473,9 @@ def load_snapshot(self, handle):
         obj.framework = self
         obj.handle = handle
         obj.restore(data)
+        if handle.path in self._objects:
+            raise RuntimeError(f"two objects claiming to be {handle.path} have been created")
+        self._objects[handle.path] = obj
```

Comment:
Shouldn't this be a call to `self._track(obj)`?

Reply:
yep, I caught the one in __init__ but forgot to update the one in load_snapshot.